Source/NSCollectionView.m

```diff
@@ -26,37 +29,81 @@ Frank LeGrand (frank.legrand@testplant.com)
 */
 
 #import "Foundation/NSKeyedArchiver.h"
+
 #import <Foundation/NSGeometry.h>
 #import <Foundation/NSIndexSet.h>
+#import <Foundation/NSSet.h>
+#import <Foundation/NSDictionary.h>
+#import <Foundation/NSMapTable.h>
 #import <Foundation/NSKeyedArchiver.h>
 
 #import "AppKit/NSApplication.h"
 #import "AppKit/NSClipView.h"
 #import "AppKit/NSCollectionView.h"
 #import "AppKit/NSCollectionViewItem.h"
+#import "AppKit/NSCollectionViewLayout.h"
+#import "AppKit/NSCollectionViewGridLayout.h"
 #import "AppKit/NSEvent.h"
 #import "AppKit/NSGraphics.h"
 #import "AppKit/NSImage.h"
 #import "AppKit/NSKeyValueBinding.h"
+#import "AppKit/NSNib.h"
 #import "AppKit/NSPasteboard.h"
 #import "AppKit/NSWindow.h"
+
 #import "GSGuiPrivate.h"
+#import "GSFastEnumeration.h"
 
 #include <math.h>
 
 static NSString* NSCollectionViewMinItemSizeKey              = @"NSMinGridSize";
```

Comment:
Don't all these values now require APPKIT_DECLARE?

Reply:
Actually, I am going to move them to externs.m.  I think they belong there.